services/graph/pkg/unifiedrole/unifiedrole.go

```diff
@@ -258,6 +284,48 @@ func NewSecureViewerUnifiedRole() *libregraph.UnifiedRoleDefinition {
 	}
 }
 
+// NewFederatedViewerUnifiedRole creates a federated viewer role
+func NewFederatedViewerUnifiedRole() *libregraph.UnifiedRoleDefinition {
+	r := conversions.NewViewerRole()
+	return &libregraph.UnifiedRoleDefinition{
+		Id:          proto.String(UnifiedRoleFederatedViewerID),
+		Description: proto.String("View and download."),
```

Comment:
Shouldn't this description be adjusted so we can differentiate between a federated users and a non federated user?